server/routes/index.js

```diff
@@ -2,12 +2,14 @@ import express from 'express';
 import authRoutes from './auth.routes';
 import userRoutes from './users.routes';
 import articleRoutes from './article.routes';
+import reportCategories from './report-categories.routes';
 
 const router = express.Router();
 
 router.use('/auth', authRoutes);
 router.use('/users', userRoutes);
 router.use('/article', articleRoutes);
+router.use('/reportCategories', reportCategories);
```

Comment:
Hi @cupidcoder I think this can be better presented with `/report/categories`

Reply:
@Chiazokam I agree, I would implement right away

Reply:
Did you implement this @cupidcoder ?

Reply:
@Chiazokam @paradisekelechi I have done this. Thanks